<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AccountCDLDialog</class>
 <widget class="QDialog" name="AccountCDLDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>381</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="createCheck">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="deleteCheck">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>60</y>
     <width>99</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Check one, then fill in required fields</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>116</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordLine">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string extracomment="password"/>
   </property>
   <property name="placeholderText">
    <string>password</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="passwordCheck">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Change Password</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AccountCDLDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AccountCDLDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>296</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>createCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>AccountCDLDialog</receiver>
   <slot>createCheckStateHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>14</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>AccountCDLDialog</receiver>
   <slot>deleteCheckStateHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>174</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>34</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>passwordCheck</sender>
   <signal>toggled(bool)</signal>
   <receiver>AccountCDLDialog</receiver>
   <slot>passwordCheckStateHandler()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>92</y>
    </hint>
    <hint type="destinationlabel">
     <x>54</x>
     <y>92</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>createCheckStateHandler()</slot>
  <slot>deleteCheckStateHandler()</slot>
  <slot>lockCheckStateHandler()</slot>
  <slot>passwordCheckStateHandler()</slot>
 </slots>
</ui>
